pfs/lineLists/skyLines.txt

```diff
@@ -1,1769 +1,655 @@
-# lambda     Intensity  Species  Status
-# nm,vacuum
+# Columns:
+# 1: wavelength (nm)
+# 2: intensity (arbitrary units)
+# 3: description (ionic species)
+# 4: status (bitmask)
```

Comment:
Would be useful to have an additional column for comments, notes, references etc. I see the latter have been removed.

Reply:
I don't think there has ever been another column. You can add whatever you want after a comment character.

Reply:
You are absolutely right - RHL and I introduced additional information in skylines.txt, but only in the form of comments, not an additional column. Those comments have been removed, but I can re-introduce them later if necessary.